Automattic/vip-go-nextjs-skeleton
# Add redis implementation to the skeleton

This adds a redis example implementation to the skeleton. It uses a generic weather API to get information and cache them for future use.

## Test
1. Get the branch
2. Start a local redis instance
3. Go to `.env` and expose your redis URL like: `VIP_REDIS_PRIMARY="127.0.0.1:6379"`
4. Go to `http://localhost:3000/api/weather?city=miami`
5. Verify the response has a `"source":"api"`
6. Refresh the page
7. Verify the response has a `"source":"cache"`
8. Wait for 10 seconds or more and refresh the page
9. The response should have a `"source":"api"` (cache destruction after 10 sec)
10. Enjoy
